### PR TITLE
Fix IO::Spec::* parent class

### PR DESCRIPTION
**Pull request by stoned**
While here uniformize introduction to these classes

### CODE REVIEW

File: doc/Type/IO/Spec/QNX.pod6 [hunk 1]
```diff
@@ -4,18 +4,19 @@
...
 
-This sub-class of L<IO::Spec|/type/IO::Spec> specific to QNX will be
-available via C<$*SPEC> if the perl is running on QNX.
+An object of this type is available via the variable C<$*SPEC> if the
+Raku interpreter is running on a C<QNX> platform.
 
-B<NOTE:> the C<IO::Spec::*> classes provide low-level path operations. Unless
-you're creating your own high-level path manipulation routines, you don't
-need to use C<IO::Spec::*>. Use L«C<IO::Path>|/type/IO::Path» instead.
+The C<IO::Spec::*> classes provide low-level path operations. Unless
```
Comment (JJ) [inline doc/Type/IO/Spec/QNX.pod6:12]: Don't like repetition of text in the documentation. As long as you're improving this, maybe you should put all this text in a single place and link it.

**Review by JJ — CHANGES_REQUESTED** [review]
Can you please address the minor concerns indicated in the review? Thanks again for your work.

File: doc/Type/IO/Spec/Cygwin.pod6 [hunk 2]
```diff
@@ -4,18 +4,19 @@
 
 =SUBTITLE Platform specific operations on file and directory paths for Cygwin
 
-    class IO::Spec::QNX is IO::Spec { }
+    class IO::Spec::QNX is IO::Spec::Unix { }
```
[the remark this answers is not in the document]
Comment (stoned) [reply doc/Type/IO/Spec/Cygwin.pod6:7]: Yes ! Indeed, and that is what I wanted to fix in the first place! :)

File: doc/Type/IO/Spec/QNX.pod6 (hunk 1 above)
Comment (stoned) [reply doc/Type/IO/Spec/QNX.pod6]: Sure, what about the update I pushed ? Would that be OK ?

**Review by JJ — APPROVED** [review]
:heart:  it. Thanks!

**Pull request merged**